bundles/automation/org.eclipse.smarthome.automation.provider.file/src/main/java/org/eclipse/smarthome/automation/internal/provider/file/AbstractFileProvider.java

```diff
@@ -135,7 +135,7 @@ public void importResources(File file) {
             File[] files = file.listFiles();
             if (files != null) {
                 for (File f : files) {
-                    if (!file.getName().startsWith(".")) {
+                    if (!file.isHidden()) {
```

Comment:
With respect to my other comment, we could ensure that the method is not called for hidden files or we could move the `file.isHidden()` check to the beginning of the method, to skip hidden directories, too.